socorro/unittest/processor/test_signature_utilities.py

```diff
@@ -189,205 +128,114 @@ def test_normalize_with_collapse_args(self):
     #--------------------------------------------------------------------------
     def test_generate_1(self):
         """test_generate_1: simple"""
-        for s, c in (self.setup_config_C_sig_tool('a|b|c', 'd|e|f'),
-                     self.setup_db_C_sig_tool('a|b|c', 'd|e|f')):
-            a = [x for x in 'abcdefghijklmnopqrstuvwxyz']
-            e = 'd | e | f | g'
-            sig, notes = s.generate(a)
-            self.assert_equal_with_nicer_output(e, sig)
-
-            a = [x for x in 'abcdaeafagahijklmnopqrstuvwxyz']
-            e = 'd | e | f | g'
-            sig, notes = s.generate(a)
-            self.assert_equal_with_nicer_output(e, sig)
+        s, c = self.setup_config_C_sig_tool(['a', 'b', 'c'], ['d', 'e', 'f'])
+        a = [x for x in 'abcdefghijklmnopqrstuvwxyz']
```

Comment:
These lines are weird. Is this just to convert the string into a list? If so, strings are iterables, so you can just do:

```
a = list('abcdefgh...')
```